app/models/mixins/vmdb_database_metrics_mixin.rb

```diff
@@ -9,7 +9,7 @@ def first_or_last_capture(interval_name, sort_order)
     my_metrics
       .where(:capture_interval_name => interval_name)
       .select(:timestamp)
-      .order("timestamp #{sort_order}")
+      .order(sort_order == "DESC" ? "timestamp DESC" : "timestamp ASC")
```

Comment:
I'm trying to get away from the `"timestamp #{sort_order}"` -- which has potential for "sql injection"

And to tell you the truth, this should be a `min/max` of `timestamp`, which I'll put a PR at a later date